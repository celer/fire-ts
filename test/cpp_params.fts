<%for(var i in params){%>
arg<%=i%> = getArgument<<%=params[i].type%>>(args[<%=i%>]);
<%}%>
<%for(var i in params){%>
arg<%=i%> = getArgument<<%=params[i].type%>>(args[<%=i%>]);
<%}%>
<%params2.map(function(i){%>
arg<%=i%> = getArgument<>(args[<%=i%>]);
<%});%>

<%for(var i in params){%>
		arg<%=i%> = getArgument<<%=params[i].type%>>(args[<%=i%>]);
<%}%>
<%for(var i in params){%>
		arg<%=i%> = getArgument<<%=params[i].type%>>(args[<%=i%>]);
<%}%>
<%params2.map(function(i){%>
		arg<%=i%> = getArgument<>(args[<%=i%>]);
<%});%>
