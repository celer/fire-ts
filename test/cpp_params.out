arga = getArgument<int>(args[a]);
argb = getArgument<float>(args[b]);
arga = getArgument<int>(args[a]);
argb = getArgument<float>(args[b]);
arga = getArgument<>(args[a]);
argb = getArgument<>(args[b]);

	arga = getArgument<int>(args[a]);
	argb = getArgument<float>(args[b]);
	arga = getArgument<int>(args[a]);
	argb = getArgument<float>(args[b]);
	arga = getArgument<>(args[a]);
	argb = getArgument<>(args[b]);
